test/extended/syncer/syncer.go

```diff
@@ -0,0 +1,79 @@
+package syncer
+
+import (
+	"os"
+	"strings"
+	"time"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	apb "github.com/kcp-dev/kcp-tests/test/extended/apibinding"
+	exutil "github.com/kcp-dev/kcp-tests/test/extended/util"
+)
+
+var _ = g.Describe("[area/transparent-multi-cluster]", func() {
+	defer g.GinkgoRecover()
+
+	var (
+		k = exutil.NewCLIWithWorkSpace("kcp-syncer")
+	)
+
+	g.It("Author:pewang-Critical-[Smoke][BYO] Validate create, modifying, deleting a deployment from KCP gets synced to the pcluster", func() {
+		pclusterKubeconfig := os.Getenv("PCLUSTER_KUBECONFIG")
+		if pclusterKubeconfig == "" {
+			g.Skip("None pcluster kubeconfig set for the test scenario")
+		}
+		myWs := k.WorkSpace()
+		k.SetGuestKubeconf(pclusterKubeconfig)
+
+		g.By("# Create workload sync and generate syncer resources manifests in current workspace")
+		mySyncer := NewSyncTarget()
```

Comment:
@Phaow where do we mention the syncer image that we would like to work with while generating the synctarget output file ?